npm/private/lifecycle/lifecycle-hooks.js

```diff
@@ -169,17 +210,23 @@ async function copyRecursive(src, dest) {
     } catch (e) {
         // Note: use .log rather than .error. The former is deferred and the latter is immediate.
         // The error is harder to spot and parse when it appears in the middle of the other logs.
-        if (e.code === "ELIFECYCLE" && !!e.pkgid && !!e.stage && !!e.script) {
```

Comment:
Is this block all just formatting? What's causing this?

Reply:
It's the formatter. Guessing that who ever changed this file last didn't have the precommit hooks in place & we don't check formatting on CI presently.